Lil-Peep/src/styles/modules/pdp.scss

```diff
@@ -0,0 +1,113 @@
+.product-container {
+  background-color: #f7f8f9;
+  flex-direction: column;
+  height: 500px;
+
+  @media #{$breakpoint-tablet} {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+  }
+}
+
+.half-left {
+  display: flex;
+  justify-content: center;
+
+  @media #{$breakpoint-tablet} {
+    align-items: center;
+    display: flex;
+    height: 100%;
+    width: 50%;
+  }
+}
+
+.super-product {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  .box-product {
+    display: flex;
+    height: 300px;
+    justify-content: center;
+    width: 100%;
+
+    .product {
+      height: 100%;
+      justify-content: center;
+      object-fit: contain;
+      width: 100%;
+    }
+
+    @media #{$breakpoint-tablet} {
+      width: 100%;
+    }
+  }
+
+  .box-micro {
+    display: flex;
+    justify-content: center;
+    padding-top: 25px;
+    width: 70%;
+
+    a {
+      height: 85px;
+      margin-right: 20px;
+      width: 85px;
+
+      img {
+        border-color: transparent;
+        border-style: solid;
+        border-width: 5px;
+        object-fit: contain;
+      }
+
+      &:last-child {
+        margin-right: 0px;
+        margin-top: 0px;
+      }
+
+      @media #{$breakpoint-tablet} {
+        height: auto;
+        max-width: 100%;
+      }
+    }
+
+    .selected {
+      border-color: #4a90e2;
```

Comment:
Color literals like `#4a90e2` should only be used in variable declarations; they should be referred to via variable everywhere else.